### DISCUSSION TITLE
Full cache strategy and DigitalOcean App Platform

### DISCUSSION THREAD

**Discussion by scottmetoyer** [statamic/cms#9234] [Q&A]
Has anyone successfully configured DigitalOcean App Platform to run a Statamic site with the full cache strategy?

Specifically, how did you set up the required rewrite rules? I see that [you can set up rewrite rules through the web settings,](https://docs.digitalocean.com/products/app-platform/how-to/url-rewrites/) but that doesn't seem to support the [rules that are required for Statamic full cache](https://statamic.dev/static-caching#server-rewrite-rules). Anyone got this working?

**Comment by duncanmcclean** [comment]
It doesn't look like DigitalOcean App Platform gives you access to the Nginx/Apache configuration for applying the relevant rewrite rules for **full measure** static caching.

Instead, you could potentially look at using [**half measure**](https://statamic.dev/static-caching#application-driver) static caching which doesn't require configuring any rewrite rules.
